tests/integration/test_cmorizer.py

```diff
@@ -143,29 +146,77 @@ def arguments(*args):
     sys.argv = backup
 
 
-def test_cmorize_obs_woa_no_data(tmp_path):
+@pytest.mark.skipif(
+    version.parse(esmvalcore.__version__) >= version.parse("2.14.0"),
+    reason='ESMValCore >= v2.14.0',
+)
+def test_cmorize_obs_woa_no_data_config_file(tmp_path):
     """Test for example run of cmorize_obs command."""
+    config_file = write_config_file(tmp_path)
+    os.makedirs(os.path.join(tmp_path, 'raw_stuff', 'Tier2'))
+    os.makedirs(os.path.join(tmp_path, 'output_dir'))
+    with keep_cwd():
+        with pytest.raises(Exception):
```

Comment:
This should be avoided, see https://docs.astral.sh/ruff/rules/assert-raises-exception/

Reply:
Changed in https://github.com/ESMValGroup/ESMValTool/pull/3761/commits/697e60ffdfd17c1fdd14312f16cb95b203796f68